packages/transition-backend/src/models/db/transitNodes.db.queries.ts

```diff
@@ -94,7 +94,6 @@ const geojsonCollection = async (
             'name', name,
             'description', description,
             'geography', ST_AsGeoJSON(geography)::jsonb, 
-            'data', data,
```

Comment:
Is the // TODO above related to this field or to "data" in general??

Reply:
It refers the the content in general, we may have a bit too much data still, but that saves a lot already

Reply:
The TODO refers to https://github.com/chairemobilite/transition/issues/408